docs/design/ovs-pipeline.md

```diff
@@ -406,42 +406,54 @@ In the future this table may support an additional mode of operations, in which
+### AntreaPolicyEgressRuleTable (45)
 
-For these two tables, you will need to keep in mind the ACNP
+For this table, you will need to keep in mind the ACNP
 [specification](#antrea-networkpolicy-crd-implementation)
 that we are using.
 
-These two tables are used to implement the egress rules across all Antrea-native policies.
-Depending on the tier to which the ACNP or ANP belongs to, the rules will be 
-installed in a table corresponding to that tier. The egress table to tier mappings 
-is as follows:
-
+This table is used to implement the egress rules across all Antrea-native policies,
+except for policies that are created in the Baseline Tier. Antrea-native policies
+created in the Baseline Tier will be enforced after K8s NetworkPolicies, and their
+egress rules are installed in the [EgressDefaultTable] and [EgressRuleTable]
+respectively, i.e.
 ```text
-Application tier -> DefaultTierEgressRuleTable(49)
-All other tiers  -> MultiTierEgressRuleTable(45)
+Baseline Tier     ->  EgressDefaultTable(60)
+K8s NetworkPolicy ->  EgressRuleTable(50)
+All other Tiers   ->  AntreaPolicyEgressRuleTable(45)
 ```
 
 Since the example ACNP resides in the Application tier, if you dump the flows for
-table 49, you should see something like this:
+table 45, you should see something like this:
 
 ```text
-1. table=49, priority=64990,ct_state=-new+est,ip actions=resubmit(,70)
-2. table=49, priority=58600,conj_id=2,ip actions=load:0x2->NXM_NX_REG6[],resubmit(,70)
-3. table=49, priority=58600,ip,nw_src=10.10.1.6 actions=conjunction(2,1/3)
-4. table=49, priority=58600,ip,nw_dst=10.10.1.8 actions=conjunction(2,2/3)
-5. table=49, priority=58600,udp,tp_dst=53 actions=conjunction(2,3/3)
-6. table=49, priority=0 actions=resubmit(,50)
+1. table=45, priority=64990,ct_state=-new+est,ip actions=resubmit(,61)
+2. table=45, priority=14000,conj_id=1,ip actions=load:0x1->NXM_NX_REG5[],ct(commit,table=61,zone=65520,exec(load:0x1->NXM_NX_CT_LABEL[32..63]))
+3. table=45, priority=14000,ip,nw_src=10.10.1.6 actions=conjunction(1,1/3)
+4. table=45, priority=14000,ip,nw_dst=10.10.1.8 actions=conjunction(1,2/3)
+5. table=45, priority=14000,udp,tp_dst=53 actions=conjunction(1,3/3)
+6. table=45, priority=0 actions=resubmit(,50)
 ```
+Notice how we use the OVS built-in `conjunction` action to implement policies
+efficiently. This enables us to do a conjunctive match across multiple
+dimensions (source IP, destination IP, port) efficiently without "exploding" the
+number of flows. By definition of a conjunctive match, we have at least 2
+dimensions. For our use-case we have at most 3 dimensions.
+
+The only requirements on `conj_id` is for it to be a unique 32-bit integer
+within the table. At the moment we use a single custom allocator, which is
+common to all tables that can have NetworkPolicy flows installed (45, 50,
+60, 85, 90 and 100). This is why `conj_id` is set to 1 in the above example
+(2 was allocated for the egress rule of our K8s NetworkPolicy example).
```

Comment:
This paragraph made sense back when `conj_id` was set to 2 for the first table, as it explained why the `conj_id` used was 2 and not 1.
With the change in example flows, it makes more sense IMO to keep this information where it was (EgressRuleTable) and rephrase it appropriately.

Reply:
Sure, changed it back.